rxfingerprint/src/main/java/com/mtramin/rxfingerprint/CipherProvider.java

```diff
@@ -88,6 +89,26 @@ Cipher getCipherForEncryption() throws IOException, GeneralSecurityException {
 		}
 	}
 
+	// https://github.com/googlesamples/android-FingerprintDialog/issues/21
+	// https://issuetracker.google.com/issues/65578763
+	@TargetApi(Build.VERSION_CODES.M)
+	Exception mapCipherFinalOperationException(Exception e) {
+		boolean shouldThrowKeyPermanentlyInvalidatedException = invalidatedByBiometricEnrollment &&
+				Build.VERSION.SDK_INT >= 26 /*Build.VERSION_CODES.O*/ &&
```

Comment:
As reported in the issue on the Google issue tracker it seems that the newest Android developer preview contains a fix for this issue. I think we should limit this to SDK 26 as the new developer preview is SDK 27.

Reply:
Agree.